server/service/src/processors/transfer/requisition/create_response_requisition.rs

```diff
@@ -138,8 +138,7 @@ fn generate_response_requisition_lines(
                  available_stock_on_hand,
                  average_monthly_consumption,
                  snapshot_datetime,
-                 // TODO should comment transfer ?
-                 comment: _,
+                 comment,
```

Comment:
The mSupply version adds in the request requisition number etc. We aren't going to do that?

```
[requisition]comment:=Get indexed string(17001; 175)+$t_serial+Choose([requisition]comment=""; ""; " ("+[requisition]comment+")")  // "From request requisition N (comment - if any)"
	[requisition]requester_reference:=Get indexed string(17001; 175)+$t_serial+Choose([requisition]requester_reference=""; ""; " ("+[requisition]requester_reference+")")  // "From request requisition N (requester_reference - if any)"
```

Reply:
Which field would correspond to the request requisition number? The requisition id? 🤔

Reply:
the `requisition_number` I think - since that's an incrementing int it's a little easier to read.